docs/misp2_installation_manual_18.04.md

```diff
@@ -131,6 +131,14 @@ Add the MISP2 repository’s signing key to the list of trusted keys:
 wget -qO - https://artifactory.niis.org/api/gpg/key/public | apt-key add -
 ```
 
+**NB!** Due to a known issue in the installation package, please perform the
+following action after upgrading your MISP2 installation:
+
+* Open the file `/var/lib/tomcat8/webapps/misp2/WEB-INF/classes/config.cfg`
+* Uncomment the line `mobileID.rest.trustStore.path =
+  MOBILE_ID_TRUST_STORE_PATH` by removing the `#` symbol from the beginning of
+  the line (the value can remain as is)
+
```

Comment:
I would move this to its own sub chapter "4.1.1 Doing a version upgrade" that's located right before "4.2 MISP2 database package". In that case, the table of contents should be updated too.

### 4.1.1 Doing a version upgrade

Due to a known issue in the installation package, please perform the following action after upgrading your MISP2 installation:

* Open the file `/var/lib/tomcat8/webapps/misp2/WEB-INF/classes/config.cfg`.
* Uncomment the line `mobileID.rest.trustStore.path =
  MOBILE_ID_TRUST_STORE_PATH` by removing the `#` symbol from the beginning of
  the line (the value can remain as is).